gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/model/GbGradebookData.java

```diff
@@ -619,8 +624,8 @@ public boolean isNull() {
 			return score == null;
 		}
 
-		public boolean isLarge() {
-			return score != null && Double.valueOf(score) >= 16384;
+		public boolean isPackable() {
+			return isNull() || (Double.valueOf(score) >= 0 && Double.valueOf(score) < 16384);
```

Comment:
<= ?

Reply:
I checked and rechecked, and think I got this right :)  16384 is the smallest number that *can't* be represented in the binary packed format (we only use 2 bytes per number and reserve the top 2 bits for type codes, so the highest we can represent is 16383)

Reply:
yup makes sense to me!